USSF-ORBIT/ussf-portal-client
# fix: Production Dockerfile

## Description 

Updates to the production Dockerfile: using `slim` instead of alpine/distroless because of Keystone/Prisma (we'll come back to this), and fix passing env variables for building.

To test:
```
docker build -t <image tag> . --build-arg NEXT_PUBLIC_SITE_URL=${NEXT_PUBLIC_SITE_URL}
docker run -p 3000:3000 <image tag>
```

There should be no server errors in the console about Prisma when viewing My Space or Sites and Applications.
